src/utils/hooks/useConfirmationModal.tsx

```diff
@@ -79,17 +89,17 @@ const useConfirmModal = ({
   const showModal = ({
     title,
     message,
-    OKText,
+    confirmText: OKText,
     cancelText,
-    onOK,
+    onConfirm: onOK,
     onCancel,
-  }: Partial<ConfirmModalState>) => {
+  }: Partial<ConfirmModalState> = {}) => {
     const newState: Partial<ConfirmModalState> = {}
     if (title) newState.title = title
     if (message) newState.message = message
-    if (OKText) newState.OKText = OKText
+    if (OKText) newState.confirmText = OKText
     if (cancelText) newState.cancelText = cancelText
-    if (onOK) newState.onOK = onOK
+    if (onOK) newState.onConfirm = onOK
```

Comment:
hmm, not sure why OKText and onOk

Reply:
Oops - -yeah. I renamed this at one point and I guess this instance automatically aliased it rather than fully renamed it. Thanks. 799f4f2e